docs/data/hubble/analytics-platforms.mdx

```diff
@@ -7,14 +7,16 @@ title: Ecosystem Data Analytics Platforms
 
 The following is a list of data analytics platforms that make a complete historical record of Pubnet Stellar network data available.
 
-| Platform | Access | Dashboard |
-| --- | --- | --- |
-| [Dune](https://docs.dune.com/home) | [Access](https://dune.com/) | [Stellar Dashboard](https://dune.com/scoffie/stellar) |
-| [Ortege](https://www.ortege.ai/) | [Access](https://app.ortege.ai/login/) |  |
-
-## SDF Provided Data Analytics
-
-SDF provides a publicly available dataset with a complete historical record of Stellar network data available. Please see the [Analyst Guide](./analyst-guide) for more information.
+|  | SDF Provided Data Analytics (Native) | Dune | Artemis | Nansen |
+| --- | --- | --- | --- | --- |
+| Description | SDF provides a publicly available dataset with a complete historical record of Stellar network data available. | Raw dataset for the stellar network maintained by Dune. This is available in Dune portal | Aggregated dataset for the Stellar network maintained by Artemis. This is available in Bigquery as well as Artemis portal | Aggregated dataset for the Stellar network maintained by Nansen. This is available in Nansen portal |
+| Home page |  | https://dune.com/ | https://www.artemis.xyz/ | https://app.nansen.ai/ |
+| Unique feature | Raw dataset layer of the stellar network updated every 15 mins | Raw dataset layer of the stellar network updated every hour. You can compare different chains on the platform | Provides dashboard to compare stellar chain activity with ecosystem chains. | Provides dashboard to compare stellar chain activity with ecosystem chains |
+| Pricing |  |  |  |  |
```

Comment:
For pricing:
- SDF hubble - End users incur the cost to execute queries. Visit the [BigQuery Pricing Page](https://cloud.google.com/bigquery/pricing#analysis_pricing_models) to learn more
- Dune - Free for simpler queries. Need to pay for complex queries
- Artemis - Free dashboards. Need to pay for accessing raw dataset
- Nansen - Free dashboards. Need to pay for accessing raw dataset

Reply:
Yes that sounds right. And if you add Flipside, the entire data access (raw, dashboards) is free

Reply:
It's launched: 
<img width="334" alt="image" src="https://github.com/user-attachments/assets/59105fe7-c97d-4a03-a395-9d25064c73b7" />